desk/app/groups-ui.hoon

```diff
@@ -259,6 +259,21 @@
         'groups'^groups
         'contacts'^contacts
     ==
+  ::
+      [%x %v6 %changes since=@ ~]
```

Comment:
Converting to json already in groups-ui is the new way? A difference with the other approach of returning typed values and doing the final conversion in groups-ui is that this will not catch a type mistake in a scry endpoint, so we should make sure each of these has been disciplined in the respective agents.

If we decide to do that, we can consider making groups-ui consistent and converting the other endpoints to this style.

Reply:
Yeah this feels much nicer in that you don't have to construct a separate type for what is really intended to be just a collation of json scry requests.

_Every_ scry endpoint _should_ be disciplined, yes!